components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Publisher/src/Deploy/IoTHubPublisherDeployment.cs

```diff
@@ -89,12 +89,12 @@ public sealed class IoTHubPublisherDeployment : IHostProcess {
                 createOptions = _serializer.SerializeToString(new {
                     Hostname = "publisher",
                     Cmd = new[] {
-                        "PkiRootPath=/mount/pki",
-                        "--aa"
-                    },
+                    "PkiRootPath=/mount/pki",
+                    "--aa"
+                },
```

Comment:
@dacolgit, can you please revert back formatting changes in `IoTHubPublisherDeployment.cs` ? Nesting looks confusing now.